app/views/accessibility_statement.scala.html

```diff
@@ -14,71 +14,101 @@
-    <p>@Messages("fuf.accessibility.statement.accessible.awareness")</p>
-
-    <h2 class="govuk-heading-l">@Messages("fuf.accessibility.statement.whattodo.heading")</h2>
-    <p>@Html(Messages("fuf.accessibility.statement.whattodo.p1"))</p>
-
-    <ul class="list list-bullet">
-        <li>@Messages("fuf.accessibility.statement.whattodo.l1")</li>
-        <li>@Messages("fuf.accessibility.statement.whattodo.l2")</li>
-    </ul>
-
-    <h2 class="govuk-heading-l">@Messages("fuf.accessibility.statement.reporting.heading")</h2>
-    <p>@Html(Messages("fuf.accessibility.statement.reporting.p1"))</p>
-
-    <h2 class="govuk-heading-l">@Messages("fuf.accessibility.statement.enforcement.heading")</h2>
-    <p>@Html(Messages("fuf.accessibility.statement.enforcement.p1"))</p>
-
-    <h2 class="govuk-heading-l">@Messages("fuf.accessibility.statement.contacting.heading")</h2>
-    <p>@Messages("fuf.accessibility.statement.contacting.p1")</p>
-    <p>@Messages("fuf.accessibility.statement.contacting.p2")</p>
-    <p>@Html(Messages("fuf.accessibility.statement.contacting.p3"))</p>
-
-    <h2 class="govuk-heading-l">@Messages("fuf.accessibility.statement.technical.heading")</h2>
-    <p>@Messages("fuf.accessibility.statement.technical.p1")</p>
-    <p>@Html(Messages("fuf.accessibility.statement.technical.p2"))</p>
+@()(implicit request: Request[_], messages: Messages)
 
-    <h2 class="govuk-heading-l">@Messages("fuf.accessibility.statement.howtested.heading")</h2>
-    <p>@Messages("fuf.accessibility.statement.howtested.p1")</p>
-    <p>@Html(Messages("fuf.accessibility.statement.howtested.p2"))<p>
-    <p>@Messages("fuf.accessibility.statement.howtested.p3")</p>
+@govukLayout(title = Title("fuf.accessibility.statement.title")) {
+
+  @pageTitle(text = messages("fuf.accessibility.statement.title"))
+
+  @paragraphBody(message = messages("fuf.accessibility.statement.intro.p1"), classes = "govuk-body-l")
+  @paragraphBody(message = messages("fuf.accessibility.statement.intro.p2"))
+  @paragraphBody(message = messages("fuf.accessibility.statement.intro.p3"))
+
+
+  <h2 id="information-you-need" class="govuk-heading-l">@messages("fuf.accessibility.statement.using.heading")</h2>
+  @paragraphBody(message = messages("fuf.accessibility.statement.using.p1"))
+  @paragraphBody(message = messages("fuf.accessibility.statement.using.p2"))
+  @bulletList(
+    elements = Seq(
+      messages("fuf.accessibility.statement.using.l1"),
+      messages("fuf.accessibility.statement.using.l2"),
+      messages("fuf.accessibility.statement.using.l3"),
+      messages("fuf.accessibility.statement.using.l4"),
+      messages("fuf.accessibility.statement.using.l5")
+    )
+   )
+  @paragraphBody(message = messages("fuf.accessibility.statement.using.p3"))
+  @paragraphBody(message = messages("fuf.accessibility.statement.using.p4"))
+
+
+  <h2 id="information-you-need" class="govuk-heading-l">@messages("fuf.accessibility.statement.accessible.heading")</h2>
```

Comment:
The id is repeated (!)

Reply:
Does no longer apply to the current state of the PR